src/NPFinancial.jl

```diff
@@ -288,5 +288,19 @@ function mirr(values::AbstractVector{<:Real}, finance_rate::Real, reinvest_rate:
     return (numer/denom)^(1/(n - 1))*(1 + reinvest_rate) - 1
 end
 
+"""
+    eir(rate::Real, nper::Real)
+
+Computes the effective interest rate `ieff` given 
+* `rate` is the nominal (annual) rate,
+* `nper` is the number of compounding periods.
```

Comment:
Since you're putting them as bullets, let's also remove the trailing punctuations (comma and period).

Reply:
Done!